examples/SimpleGeneratorPlugin/AutomationConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+using Zeiss.PiWeb.Sdk.Import.Modules.ImportAutomation;
+
+namespace SimpleGeneratorPlugin

```

Comment:
Does it make sense to use file scoped namespace here?

Reply:
Yes I'll change the namespaces.